.github/workflows/roman_ci.yml

```diff
@@ -31,11 +31,6 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  check:
-    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v2
-    with:
-      envs: |
-        - linux: check-dependencies
```

Comment:
If we're dropping this job I think it makes sense to drop the tox environment:
https://github.com/spacetelescope/romancal/blob/8b65d0d47b828ee523736360baf5ff1c71a9e8dd/tox.ini#L25
and update the comment in the test file (if not remove the test file):
https://github.com/spacetelescope/romancal/blob/8b65d0d47b828ee523736360baf5ff1c71a9e8dd/romancal/tests/test_import.py#L2-L8